deploy/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16-alpine AS BASE
+WORKDIR /usr/src/app
+
+COPY package.json ./
+COPY package-lock.json ./
+
+ENV SERVER_URL="http://localhost:9999"
+ENV CONTEXT="/fta-fmea"
+ENV ADMIN_ONLY=false
+
+RUN npm install
+COPY ./ ./
+
+RUN REACT_APP_BASE_API_URL=${SERVER_URL}/${CONTEXT}-server \
```

Comment:
OK definitely very good step forward,  but this is still creating build-time dependencies and not runtime dependencies as we need.

I think you will understand as you will be creating `docker-compose.yml` file. I assume you did not do that, see A/C of the ticket:
> there is docker-compose file in this repository which orchestrates backend and frontend together

I suggest to try to create docker-compose.yml first and when done also tag @LaChope to review the pull request, then if needed agree on some short meeting to explain each other ideas.

Reply:
Of course, i am also available to discuss ...

Reply:
No, i didnt do that. 
Ok, thanks.